impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -420,17 +416,28 @@ std::optional<Entity> GaussianBlurFilterContents::RenderFilter(
     mip_count = 1;
   }
 
+  Entity snapshot_entity = entity.Clone();
+  snapshot_entity.SetTransform(Matrix());
```

Comment:
This will cause anything that's scaled up to be rendered at a resolution which is too low for accurate results.

Perhaps a good heuristic to use here would would be to extract the max basis length and apply it here.